examples/hif2a/fit_to_multiple_rbfes.py

```diff
@@ -155,6 +155,7 @@ def _save_forcefield(filename, ff_params):
     else:
         # Setup GRPC client
         client = GRPCClient(hosts=args.hosts)
+    client.verify()
```

Comment:
@maxentile Added you to the PR so you were aware of the (potential) change to adding a verify method to the client